x-pack/legacy/plugins/apm/cypress/cypress/support/step_definitions/apm.ts

```diff
@@ -0,0 +1,48 @@
+/// <reference types="Cypress" />
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { loginAndWaitForPage } from '../../integration/helpers';
+
+const { Given, When, Then } = require('cypress-cucumber-preprocessor/steps');
+
+Given(`a user browses the APM UI application`, () => {
+  // open service overview page
+  loginAndWaitForPage(`/app/apm#/services`);
```

Comment:
Because a `Given` represents the pre-conditions for a test, we moved the old `before` method here.